BenchmarkTool/Generators/RecordFactory.cs

```diff
@@ -11,16 +11,35 @@ public RecordFactory()
             database = Config.GetTargetDatabase();
         }
 
-        public IRecord Create(int sensorId, DateTime timestamp, float value)
+        public IRecord Create(int sensorId, DateTime timestamp, double value)
         {
             return database switch
             {
+                Constants.DatalayertsDBClass => new RecordDatalayerts(sensorId, timestamp, value),
```

Comment:
I do not see victoriametrics? is the implementation not complete?

Reply:
VM has a probem with ingesting time information via influx ine protocoll. i wrote to them, but no answer yet. i should create an official bug report as next step.

Reply:
i added this info as a comment and commented VM out.